Mathlib/Probability/Distributions/Gaussian/Basic.lean

```diff
@@ -0,0 +1,138 @@
+  continuous linear form `L : Dual ℝ E`.
+
+## References
+
+* [Martin Hairer, *An introduction to stochastic PDEs*][hairer2009introduction]
+
+-/
+
+open MeasureTheory Complex NormedSpace
+open scoped ENNReal NNReal
+
+namespace ProbabilityTheory
+
+/-- A measure is Gaussian if its map by every continuous linear form is a real Gaussian measure. -/
+class IsGaussian {E : Type*} [TopologicalSpace E] [AddCommMonoid E] [Module ℝ E]
+  {mE : MeasurableSpace E} (μ : Measure E) : Prop where
+  map_eq_gaussianReal (L : E →L[ℝ] ℝ) : μ.map L = gaussianReal (μ[L]) (Var[L; μ]).toNNReal
+
+/-- A real Gaussian measure is Gaussian. -/
+instance isGaussian_gaussianReal (m : ℝ) (v : ℝ≥0) : IsGaussian (gaussianReal m v) where
+  map_eq_gaussianReal L := by
+    rw [gaussianReal_map_continuousLinearMap]
+    simp only [integral_continuousLinearMap_gaussianReal, variance_continuousLinearMap_gaussianReal,
+      Real.coe_toNNReal']
+    congr
+    rw [Real.toNNReal_mul (by positivity), Real.toNNReal_coe]
+    congr
+    simp only [left_eq_sup]
+    positivity
+
+variable {E F : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E] [MeasurableSpace E] [BorelSpace E]
+  [NormedAddCommGroup F] [NormedSpace ℝ F] [MeasurableSpace F] [BorelSpace F]
+  {μ : Measure E} [IsGaussian μ]
+
+/-- Dirac measures are Gaussian. -/
+instance {x : E} : IsGaussian (Measure.dirac x) where
+  map_eq_gaussianReal L := by rw [Measure.map_dirac (by fun_prop)]; simp
+
+/-- A Gaussian measure is a probability measure. -/
+instance : IsProbabilityMeasure μ where
+  measure_univ := by
+    let L : Dual ℝ E := Nonempty.some inferInstance
+    have : μ.map L Set.univ = 1 := by simp [IsGaussian.map_eq_gaussianReal L]
+    simpa [Measure.map_apply (by fun_prop : Measurable L) .univ] using this
+
+lemma IsGaussian.memLp_dual (μ : Measure E) [IsGaussian μ] (L : Dual ℝ E)
+    (p : ℝ≥0∞) (hp : p ≠ ∞) :
+    MemLp L p μ := by
+  suffices MemLp (id ∘ L) p μ from this
+  rw [← memLp_map_measure_iff (by fun_prop) (by fun_prop), IsGaussian.map_eq_gaussianReal L]
+  convert memLp_id_gaussianReal p.toNNReal
+  simp [hp]
+
+@[fun_prop]
+lemma IsGaussian.integrable_dual (μ : Measure E) [IsGaussian μ] (L : Dual ℝ E) :
+    Integrable L μ := by
+  rw [← memLp_one_iff_integrable]
+  exact IsGaussian.memLp_dual μ L 1 (by simp)
+
+/-- The map of a Gaussian measure by a continuous linear map is Gaussian. -/
+instance isGaussian_map (L : E →L[ℝ] F) : IsGaussian (μ.map L) where
+  map_eq_gaussianReal L' := by
+    rw [Measure.map_map (by fun_prop) (by fun_prop)]
+    change Measure.map (L'.comp L) μ = _
+    rw [IsGaussian.map_eq_gaussianReal (L'.comp L)]
+    congr
+    · rw [integral_map (by fun_prop) (by fun_prop)]
+      simp
+    · rw [← variance_id_map (by fun_prop)]
+      conv_rhs => rw [← variance_id_map (by fun_prop)]
+      rw [Measure.map_map (by fun_prop) (by fun_prop)]
+      simp
+
+instance isGaussian_map_equiv (L : E ≃L[ℝ] F) : IsGaussian (μ.map L) :=
```

Comment:
could you register the lemma saying that `mu.map L` is gaussian iff `mu` is when  `L` is an equiv?